Data/Text/Internal/Fusion/Common.hs

```diff
@@ -395,7 +398,7 @@ intersperse c (Stream next0 s0 len) = Stream next (I1 s0) len
 
 caseConvert :: (forall s. Char -> s -> Step (CC s) Char)
             -> Stream Char -> Stream Char
-caseConvert remap (Stream next0 s0 len) = Stream next (CC s0 '\0' '\0') len
+caseConvert remap (Stream next0 s0 _) = Stream next (CC s0 '\0' '\0') unknownSize
```

Comment:
hmm unless I'm misunderstanding something, can't this be `len + maxSize $ charSize s0` ?

Reply:
Okay, so I looked at the call sites and indeed they *do* produce only one code point per input code point although there is of course nothing in the type of `caseConvert` that requires this be the case. However, given that `caseConvert` isn't exported, I suppose we can just state this as an invariant.

Reply:
On more careful reading it seems that the point I made above actually wasn't quite right: `remap` can produce up to three codepoints for every input code-point.

Reply:
Ah ok, thanks for the explanation!